### PR TITLE
make source an optional parameter

### LINKED ISSUE #4: The example in the README doesn't work without a second parameter
Something which isn't mentioned in the docs is that the function needs to take a second argument. This argument is optionally the source code or an empty string if it isn't available. As it isn't necessarily required but it is good to pass I think it should be optional. Also I think there should be an api reference in the README.

### PR DESCRIPTION
**Pull request by Markos-Th09**
Closes #4 
This PR makes source an optional parameter. Although it is good to pass it isn't required to have a proper ast.

### CODE REVIEW

File: lib/swc-to-babel.js [hunk 1]
```diff
@@ -112,7 +112,9 @@ module.exports = (node, source) => {
             if (path.type === 'ParenthesisExpression')
                 return convertParenthesisExpression(path);
             
-            if (/^(KeyValue|KeyValuePattern|AssignmentPattern)Property$/.test(path.type))
+            if (
+                /^(KeyValue|KeyValuePattern|AssignmentPattern)Property$/.test(path.type)
+            )
```
Comment (coderaiser) [inline lib/swc-to-babel.js:117]: Could you please write it in one line and add test for it?

Comment (Markos-Th09) [reply lib/swc-to-babel.js:117]: > Could you please write it in one line and add test for it?

I am not really sure how I would test it. And also my formatter really seems to screw up the code

Comment (coderaiser) [reply lib/swc-to-babel.js:117]: ESLint, just add one more test with one argument.

**Comment by coderaiser** [comment]
Nice 👍!

**Pull request merged**